var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -43,16 +43,12 @@ class Metis(Package):
     version('5.0.2', 'acb521a4e8c2e6dd559a7f9abd0468c5')
     version('4.0.3', 'd3848b454532ef18dc83e4fb160d1e10')
 
-    variant('shared', default=True,
-            description='Enables the build of shared libraries')
-    variant('debug', default=False,
-            description='Builds the library in debug mode')
-    variant('gdb', default=False, description='Enables gdb support')
+    variant('shared', default=True, description='Enables the build of shared libraries.')
```

Comment:
i think @tgamblin unified it in all packages a while ago, so i would not reformat `variant`'s it unless necessary.